bff-service/src/proxy/proxy.controller.ts

```diff
@@ -0,0 +1,42 @@
+import {
+  All,
+  CacheInterceptor,
+  Controller,
+  Get,
+  HttpException,
+  HttpStatus,
+  Req,
+  UseInterceptors,
+} from '@nestjs/common';
+import { ProxyService } from './proxy.service';
+import { Request } from 'express';
+
+@Controller()
+export class ProxyController {
+  constructor(private proxyService: ProxyService) {}
+
+  @Get('product/products')
```

Comment:
for current implementation the cache will NOT work for the path with the trailing slash

Reply:
It will not work if we will change req by adding trailing slash just once (at first req with trailing slash). For other requests with trailing slash we will receive cached result.

Reply:
@inshatan thanks for that!